lib/platform.js

```diff
@@ -1,14 +1,26 @@
 var os      = require('os');
+var config  = require('./config');
 
 var folders = {
+  osx: 'osx',
   darwin: 'osx',
   linux : 'linux',
   sunos : 'sunos'
 };
 
+exports.isSupported = function isValid(platform) {
+  // return ['osx', 'linux', 'sunos'].indexOf(platform) >= 0;
+  return folders.hasOwnProperty(platform);
+};
+
+exports.getPrefered = function getPrefered() {
+  var configuredPlaform = config.get().platform;
+  return configuredPlaform || folders[os.platform()];
```

Comment:
From what I understand, this function returns the name of the folder for the current platform. Can we rename is to `folderForPreferredPlatform` for example?

Also, should we use `folders[configuredPlaform]` so that users can specify aliases, like `--os=darwin` instead of having to specify the real folder name?